tests/kernel/wave/attention/paged_attention_test.py

```diff
@@ -251,7 +251,7 @@ def testPagedFlashDecoding(
         num_kv_splits, shape.num_seqs, shape.num_query_heads, dtype=torch.float32
     )
     output = device_zeros(
-        shape.num_seqs, shape.num_query_heads, shape.head_size_kv, dtype=torch.float32
+        shape.num_seqs, shape.num_query_heads, shape.head_size_kv, dtype=torch.float16
```

Comment:
Was this intended?

Reply:
Looks like it conflicted with https://github.com/iree-org/iree-turbine/pull/421 and now tests are failing: https://github.com/iree-org/iree-turbine/actions/runs/13004914350/job/36269993961